test/test_admin_integration.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from kafka import KafkaAdminClient
+from kafka.admin import ConfigResource, ConfigResourceType
+from test.fixtures import ZookeeperFixture, KafkaFixture
+from test.testutil import KafkaIntegrationTestCase, kafka_versions
+
+
+class TestAdminClientIntegration(KafkaIntegrationTestCase):
```

Comment:
Please do **not** use the `KafkaIntegrationTestCase` or anything that depends on `unittest`. 

It makes https://github.com/dpkp/kafka-python/pull/1196 much harder to merge.

There are quite a few `pytest`-based fixtures that I added recently, perhaps you can use one of those?

Reply:
Makes sense - I can see the admin client integration test was added after my PR was started so i'll just move my tests into the new one